src/chrome-finder.ts

```diff
@@ -141,19 +144,20 @@ export function linux() {
   ];
 
   if (process.env.LIGHTHOUSE_CHROMIUM_PATH) {
-    priorities.unshift({regex: new RegExp(`${process.env.LIGHTHOUSE_CHROMIUM_PATH}`), weight: 100});
+    priorities.unshift(
+        {regex: new RegExp(escapeRegExp(process.env.LIGHTHOUSE_CHROMIUM_PATH)), weight: 100});
   }
 
   if (process.env.CHROME_PATH) {
-    priorities.unshift({regex: new RegExp(`${process.env.CHROME_PATH}`), weight: 101});
+    priorities.unshift({regex: new RegExp(escapeRegExp(process.env.CHROME_PATH)), weight: 101});
   }
 
   return sort(uniq(installations.filter(Boolean)), priorities);
 }
 
 export function wsl() {
   // Manually populate the environment variables assuming it's the default config
-  process.env.LOCALAPPDATA = getLocalAppDataPath(`${process.env.PATH}`);
+  process.env.LOCALAPPDATA = getLocalAppDataPath(<string>process.env.PATH);
```

Comment:
```suggestion
  process.env.LOCALAPPDATA = getLocalAppDataPath(`${process.env.PATH}`);
```

this one seems necessary because PATH might not be defined, no?

Reply:
If it's really undefined, then the expression `` `${process.env.PATH}` `` would yield the literal string `"undefined"`. That is, it's the same as this:

~~~js
"undefined" === `${undefined}`;
~~~

Which would match a directory named `./undefined` if it happened to contain `/Applications/Google Chrome.app`, etc. Though why a user would have `./undefined/Applications/Google Chrome.app` in their CWD is beyond me, but it's certainly not the intended behaviour... 😉

Reply:
Right, I know how string coercion works 😉 

I have a problem with unsafely casting this because `getLocalAppDataPath` is expecting a string and a `pathArgument.match` would *throw* instead of just generating a path with an `undefined` in it, which while confusing would *at worst* generate a different priority order if users are storing important programs in directories called `undefined` compared to fatally throwing every time `PATH` is `undefined`.

In this particular case with our *current* implementation, this change has no benefits with respect to your `undefined/Applications/` concern because both will not match the `regex.exec` so `undefined` will be forced into the path regardless. All this change does is expose us to future `cannot read property X` of undefined problems.

So I'd request either
- Properly fix getLocalAppDataPath to accept `undefined` as a type and handle it accordingly
- Revert the change because those wrinkles weren't part of what you were trying to fix and you've been very kind to help out enough already :)

Reply:
Oh, right, I'm with you now. Sorry, it's late and I need sleep. 😅 (Now might be a good time to point out my TypeScript knowledge is rudimentary at best).

I've rolled it back.

Reply:
no worries, thanks! :)